development/client/package.json

```diff
@@ -3,16 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.0.0-rc.4",
+    "@apollo/client": "latest",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/classnames": "^2.2.10",
+    "@types/jest": "^26.0.4",
+    "@types/node": "^14.0.23",
+    "@types/react": "^16.9.43",
+    "@types/react-dom": "^16.9.8",
```

Comment:
Strangely in the directions to add TypeScript to a create-react-app project, it uses `--save` and not `--save-dev`: https://create-react-app.dev/docs/adding-typescript/#installation I don't know why 🤷‍♀️ *But* when adding this to the development project, I decided it wasn't a big deal because this application isn't being deployed anywhere. I'm cool to change this and as well as moving the `typescript` package to devDependencies if you'd like?

Reply:
Great point on the app not being deployed anywhere - I'm okay either way. Thanks!